theano/tensor/sharedvar.py

```diff
@@ -59,11 +59,17 @@ class ScalarSharedVariable(_tensor_py_operators, SharedVariable):
 
 
 @shared_constructor
-def scalar_constructor(value, name=None, strict=False, allow_downcast=None):
+def scalar_constructor(value, name=None, strict=False, allow_downcast=None,
+                       borrow=False, ):
     """SharedVariable constructor for scalar values. Default: int64 or float64.
 
     :note: We implement this using 0-d tensors for now.
 
+    :note: We ignore the borrow parameter as we convert value to an
+      ndarray (this is a new object). This respect the semantic of
+      borrow, as it is an hint to Theano that we can reuse it, this
```

Comment:
it is an hint -> it is a hint

I would also drop the `this isn't mandatory` part since it feel like useless repetition.